vnext/Desktop.UnitTests/BaseWebSocketTests.cpp

```diff
@@ -65,7 +65,7 @@ TEST_CLASS(BaseWebSocketTest){
       return make_error_code(errc::state_not_recoverable);
     });
 
-    ws->Connect({}, {});
+    ws->Connect("ws://localhost", {}, {});
```

Comment:
>"ws://localhost" [](http://example.com/codeflow?start=16&length=16)

(nit) can you make it a shared string const, and just reference it in all these places